src/constants/icons/.icons.ts

```diff
@@ -606,6 +606,12 @@ export const pause: SVGAttributes[] = [
     d: 'M11 15.778c0 .122-.224.222-.5.222-.245 0-.45-.079-.492-.182l-.008-.04V8.222c0-.123.224-.222.5-.222.245 0 .45.079.492.182l.008.04v7.556ZM14 15.778c0 .122-.224.222-.5.222-.245 0-.45-.079-.492-.182l-.008-.04V8.222c0-.123.224-.222.5-.222.245 0 .45.079.492.182l.008.04v7.556Z',
   },
 ];
+export const phone: SVGAttributes[] = [
+  {
+    'fill-rule': 'evenodd',
```

Comment:
Is this attribute useful ? sometimes they are kept in the SVG file for no reason

Reply:
Yes, removing it breaks the svg sadly